### PR TITLE
CNV-12620: updating virt metrics docs for clarity

### PR DESCRIPTION
**Pull request by ousleyp**
- [CNV-12620](https://issues.redhat.com/browse/CNV-12620)
- cherrypick to 4.8 and 4.9
- This PR addresses the bugs identified in [CNV-12620](https://issues.redhat.com/browse/CNV-12620), fixes a couple of minor issues that I noticed along the way, and includes a few more changes intended to improve clarity/structure.
- [Preview build](https://deploy-preview-35732--osdocs.netlify.app/openshift-enterprise/latest/virt/logging_events_monitoring/virt-prometheus-queries?utm_source=github&utm_campaign=bot_dp)

### CODE REVIEW

**Review by dbasunag — APPROVED** [review]

File: modules/virt-querying-metrics.adoc [hunk 1]
```diff
@@ -5,105 +5,114 @@
...
+.Example vCPU wait time query
 [source,promql]
 ----
-topk(3, sum by (name, namespace) (round(irate(kubevirt_vmi_vcpu_wait_seconds[6m]), 0.1))) > 0
+topk(3, sum by (name, namespace) (round(irate(kubevirt_vmi_vcpu_wait_seconds[6m]), 0.1))) > 0 <1>
 ----
-The above query returns the top 3 VMs waiting for I/O at every given moment in time over the time period.
+<1> This query returns the top 3 VMs waiting for I/O at every given moment over a six-minute time period.
+
+[NOTE]
```
Comment (sjhala-ccs) [inline modules/virt-querying-metrics.adoc:32]: Because this note is kind of a prerequisite, would it be more useful to place it above the example query?

**Review by sjhala-ccs — COMMENTED** [review]
One suggestion, otherwise LGTM!

**Pull request merged**